integration-cli/cli/build/build.go

```diff
@@ -0,0 +1,30 @@
+package build
+
+import (
+	"strings"
+
+	icmd "github.com/docker/docker/pkg/testutil/cmd"
+)
+
+// WithDockerfile specify a dockerfile string to be used as Dockerfile for build operations
```

Comment:
Will this package only have "operators", or also other functionality? If just operators, wondering if it should be named different, but again, thinking out loud